ccf_transaction_processor/etc/cchost.toml

```diff
@@ -2,11 +2,11 @@
 enclave-file = "${home}/ccf/lib/libpdoenc.virtual.so"
 enclave-type = "virtual"
 consensus = "raft"
-rpc-address = "${host_address}:6006"
-node-address = "${host_address}:6007"
+rpc-address = "${host_address}:6600"
```

Comment:
Thank you for the note. This was done intentionally to enable deploying multiple nodes (though not part of this PR), so that rpc-address could be 6600 + __count__ (6006 and 6007 would have caused an issue, so    a change was needed, thought that i will use 6600 since pdo services also end with '00' for the first one, and then increments.

Reply:
hmm, i have to admit i'm here with Mic, at least from the discussion i have followed on the multi-host PR:  there is no good reason to run multiple _pdo/ccf_ nodes on a single host, as you would do it distributed (and probably managed via docker or alike, in which case you still can run it on a single host).  Of course you might have _other ccf apps_, but they would run with completely different ports?   That said, while having ports close to each other is convenient, it is certainly manageable to deal also with 6600 and 7700 :-)

Reply:
we will surely have a discussion on multi-node set up before I code up that PR, but in my mind, there are several good reasons to have that set up available in the repo. but happy to chat later after the docker pr. Thank you.

Reply:
hmm, so we could also post-pone the port change until we have that discussion ... ;-)